bullet_train-outgoing_webhooks/app/controllers/account/webhooks/outgoing/endpoints_controller.rb

```diff
@@ -60,6 +63,32 @@ def destroy
     end
   end
 
+  # POST /account/webhooks/outgoing/endpoints/:id/activate
+  def activate
+    respond_to do |format|
+      if @endpoint.update(deactivated_at: nil, deactivation_limit_reached_at: nil, consecutive_failed_deliveries: 0)
```

Comment:
This should probably be a method in the Deactivateable concern, like `#reactivate!`